.github/workflows/prettier.yml

```diff
@@ -0,0 +1,24 @@
+name: Prettier Markdown
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - 'docs/**'
+
+jobs:
+  pretty:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+      - run: npm install -g yarn
+      - run: yarn install
```

Comment:
Bit worried this could cause lockfile changes to be commited as well, perhaps add a `--frozen-lockfile` just in case? Tbh I think it's work using the cached node_modules install tho, see e.g. e2e.yaml for an easy to copy yarn setup block

Reply:
added --frozen-lockfile